src/components/molecules/SelectFilter.tsx

```diff
@@ -0,0 +1,61 @@
+import styled from 'styled-components';
+import { ReactComponent as MoreVectorSvg } from '../../assets/images/more_vector.svg';
+
+interface SelectFilterProps {
+  defaultoption: 'latest' | 'like' | 'old' | 'grade';
+  onSelectChange: (selectoption: SelectFilterProps['defaultoption']) => void;
+}
+
+const SelectFilter = ({ defaultoption, onSelectChange }: SelectFilterProps) => {
+  const handleSelectChange = (e: React.FormEvent<HTMLSelectElement>) => {
+    const selectedOption = e.currentTarget
+      .value as SelectFilterProps['defaultoption'];
+    onSelectChange(selectedOption);
+  };
+  const OPTIONS = [
+    { value: 'latest', name: '최신순' },
+    { value: 'like', name: '좋아요 순' },
+    { value: 'old', name: '오래된 순' },
+    { value: 'grade', name: '평점순' },
+  ];
```

Comment:
이거 타입을 고의적으로 명시해주면 더 타입 추론이 잘될거야
위의 리뷰를 반영했다는 가정하에
(웹에서 바로 짜는 코드라 문법이 이게 맞나 싶긴 한데)

```suggestion
  const OPTIONS: { value: Option, name: string }[] [
```

Reply:
그리고 상수는 react component 바깥으로 빼는게 더 performance 가 좋아

이러면 리렌더링 될 때마다 다시 상수를 만들어낼거니까